ci/kokoro/release/publish.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+# Record the current commit hash.
+git -C github/google-cloud-cpp log -1 --format=%H > commit-hash.txt
```

Comment:
It is probably hard to see:  the robots want no spaces between `>` and `commit-hash.txt`.

Reply:
Thanks.  Yep, I hadn't figured that out.